### ISSUE TITLE
Transparency

### ISSUE THREAD

**Issue by acrostich** [amitv87/PiP#15]
It would useful if you could add support for the transparency/opaqueness of the window so that when using as an overlay (as per the example in this article: https://exposingtheinvisible.org/en/guides/geolocating-images-suncalc/) one could control how much of the overlaid window can be seen through the PiP window.

**Comment by amitv87** [comment]
Any PiP window can be made transparent/opaque by using the only visible slider in right click menu.
I guess you are looking for this feature.

**Comment by acrostich** [comment]
@amitv87 thank you very much. This is exactly the feature I was looking for. I hadn't realised this was a slider and that it was controlling opacity.

Then my request should be on making this feature more discoverable. At least it should be listed in the Features section in the readme (also I haven't yet figured out what is the advantage of selecting the native PiP option). At best the UI should label it as such.

**Comment by amitv87** [comment]
[Added](https://github.com/amitv87/PiP/commit/0f938983961e8aa59c875eb55cd1e695e5b6f5ba) it to the feature list 

The native PiP integrates well with system panels (try opening notification panel with/without native PiP) and few other scenarios like workspace switching, snapping/collapsing to the screen edges etc.

**Comment by acrostich** [comment]
Thank you for having so promptly updated the readme and for your explanations... and obviously for having written PiP in the first place.

I just tried PiP with workplace switching and indeed this needs native PiP. This does however begs the question as to what advantage the non native PiP has. The one I noticed is the opacity slider. Again, this could be in the readme to make it obvious to all users.
